workflows/reference_files/2.4/reference_files.smk

```diff
@@ -381,3 +381,37 @@ rule get_mutect2_small_exac:
             &&
         tabix {output.vcf}
         """)
+
+
+##### SigProfiler #####
+
+rule install_sigprofiler_matrix_generator:
+    output:
+        complete = "downloads/sigprofiler_prereqs/matrix_generator.installed"
+    conda: CONDA_ENVS["sigprofiler"]
+    shell:
+        "pip install SigProfilerMatrixGenerator && touch {output.complete}"
+
+rule install_sigprofiler_genome:
+    input:
+        str(rules.install_sigprofiler_matrix_generator.output.complete)
+    output:
+        complete = "downloads/sigprofiler_prereqs/{genome_build}.installed"
+    params:
+        ref = lambda w: {"grch37":"GRCh37", "hg19":"GRCh37",
```

Comment:
Same comment regarding the additional genomes not handled here (there are likely going to be more)